### PR TITLE
Add form template for NO accident

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add form template for NO accident

Add support for minLength/maxLength in text-input.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-1507]: https://hedvig.atlassian.net/browse/GRW-1507?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #554** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/554" target="_blank">Graphite</a> 
    * **PR #556** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/556" target="_blank">Graphite</a> 
      * **PR #560** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/560" target="_blank">Graphite</a> 
        * **PR #562** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/562" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/562?utm_source=stack-comment).

File: apps/store/src/services/formTemplate/data/NO_ACCIDENT_UI.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { FormTemplateUISchema } from '../FormTemplate.types'
+
+export const NO_ACCIDENT_UI: FormTemplateUISchema = {
+  layout: {
+    sections: [
+      {
+        id: 'your-info',
+        title: { key: 'Your home' },
```
Comment (guilhermespopolin) [inline apps/store/src/services/formTemplate/data/NO_ACCIDENT_UI.ts:8]: Shouldn't be the following here?
```suggestion
        title: { key: 'Your Info' },
```

File: apps/store/src/services/formTemplate/combineFormTemplate.ts [hunk 2]
```diff
@@ -51,6 +51,11 @@ export const combineFormTemplate = ({
               if (schemaField.maximum) formField.max = schemaField.maximum
             }
 
+            if (formField.type === 'text') {
+              if (schemaField.minLength) formField.minLength = schemaField.minLength
```
[the remark this answers is not in the document]
Comment (robinandeer) [reply apps/store/src/services/formTemplate/combineFormTemplate.ts:55]: ahh nice, let's discuss if that's a lib we want to depend on!

File: apps/store/src/services/formTemplate/data/NO_ACCIDENT_UI.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { FormTemplateUISchema } from '../FormTemplate.types'
+
+export const NO_ACCIDENT_UI: FormTemplateUISchema = {
```
[the remark this answers is not in the document]
Comment (robinandeer) [reply apps/store/src/services/formTemplate/data/NO_ACCIDENT_UI.ts:3]: True that, just wanted to start as simple and flexible as possible

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/562?utm_source=stack-comment).


Job Reference: 3ZRUF3TQqrKMuSm6752V

**Review by guilhermespopolin — APPROVED** [review]

**Pull request merged**